datalad/distribution/create.py

```diff
@@ -86,14 +85,6 @@ class Create(Interface):
             doc="""enforce creation of a dataset in a non-empty directory""",
             action='store_true'),
         description=dataset_description,
-        add_to_super=add_to_superdataset,
```

Comment:
There might have been none explicitly since I could just use that option which would have discovered it, so I didn't need superdataset